src/nyc_trees/gulpfile.js

```diff
@@ -199,10 +200,19 @@ function browserifyTask(bundler) {
 }
 
 gulp.task('sass', ['clean'], function() {
-    return gulp.src('sass/main.scss')
-        .pipe(sourcemaps.init())
-        .pipe(sass({style: 'compressed'}))
-        .pipe(sourcemaps.write('./'))
+    var autoprefixProcessor = autoprefixer({ browsers: ['> 1% in US'] });
+
+    return sass('sass/main.scss', { sourcemap: true })
+        .on('error', function (err) {
+            console.error('Sass error', err.message);
```

Comment:
I tried a few things but I can't seem to get `gulp-ruby-sass` to actually pass errors through to us.  I'm going to open an issue for this for now.